x/gamm/keeper/migrate.go

```diff
@@ -109,27 +109,52 @@ func (k Keeper) GetLinkedBalancerPoolID(ctx sdk.Context, concentratedPoolId uint
 	return sdk.BigEndianToUint64(balancerPoolIdBigEndian), nil
 }
 
-// OverwriteMigrationRecords sets the balancer to gamm pool migration info to the store
-// Deletes all existing records, migrationInfo in state is completely overwitten by the given migrationInfo.
-func (k Keeper) OverwriteMigrationRecords(ctx sdk.Context, migrationInfo types.MigrationRecords) {
+// deleteMigrationKeys deletes all keys with the given prefixKey.
```

Comment:
Can you also elaborate what prefixKey can or cannot be?
```suggestion
// deleteMigrationKeys deletes all migration records with the given prefixKey.
```